### ISSUE TITLE
Logcat formatting in HTML report.

### ISSUE THREAD

**Issue by navstyachka** [gojuno/composer#34]
@artem-zinnatullin Currently I have created 4 CSS classes, check if smth is missing, screenshot is attached (if you don't like the colors I've used, just give me a screenshot with console of your taste and I will apply its colors). 

```CSS
.log__info {
  color: $green;
}

.log__debug {
  color: $electric;
}

.log__error {
  color: $red;
}

.log__warn {
  color: $yellow;
}
```
Also, there can be a problem with spaces — no matter how many spaces you add one by one, in browser thay will be trimmed into one. It can be fixed with using `&nbsp;` symbol but that can also be a problem as leads ti nowrap line.

![screenshot 2017-05-30 19 18 10](https://cloud.githubusercontent.com/assets/1816011/26593449/c12d58aa-456c-11e7-8bd1-9bd145ffae92.png)

**Comment by artem-zinnatullin** [comment]
Just to confirm, we'll need 7 log levels:

1. `default`: some sort of light grey, i.e. `#DDDDDD`
1. `verbose`: slightly darker grey than `default`, i.e. `#BBBBBB`
1. `debug`: some sort of blue, i.e. `#1D91BB` 
1. `info`: some sort of green, i.e. `#53C8A1`
1. `warning`: some sort of orange, i.e. `#DBBE10`
1. `error`: some sort of red, i.e. `#FF6B68`
1. `assert`: some sort of violet, i.e. `#C200FF`

Let's also use **background color** from Android Studio/IntelliJ Darkula theme: `#202020` so it'll look very familiar for Android Developers and QAs!

This is how it looks in my Android Studio:

<img width="769" alt="screen shot 2017-05-31 at 4 06 55 pm" src="https://cloud.githubusercontent.com/assets/967132/26633179/3318ad80-461b-11e7-81b0-bba80a29d3bc.png">

**Comment by navstyachka** [comment]
@artem-zinnatullin Used colors you suggested, here's the result:
<img width="1258" alt="screenshot 2017-06-05 00 58 03" src="https://cloud.githubusercontent.com/assets/1816011/26765697/0e2b06f4-498a-11e7-9bea-463a07612358.png">

**Comment by artem-zinnatullin** [comment]
LGTM!

**Comment by navstyachka** [comment]
```HTML
<div className="card log">
            <div className="log__default">
              05-29 15:36:04.976 4065 4477 ? LogTracker: [Rx-1-io-30] Analytics start timing event: Appearance /
              StartupTime
            </div>
            <div className="log__info">
              05-29 15:36:04.976 4065 4477 I LogTracker: [Rx-1-io-30] Analytics start timing event: Appearance /
              StartupTime
            </div>
            <div className="log__verbose">
              05-29 15:36:04.976 4065 4477 V LogTracker: [Rx-1-io-30] Analytics start timing event: Appearance /
              StartupTime
            </div>
            <div className="log__debug">
              05-29 15:36:04.977 4065 4389 D RiderApplication$initFabricAsync: [Rx-1-io-28] Google Play Services
              version: 10.2.98 (470-146496160)
            </div>
            <div className="log__error">
              05-29 15:36:04.983 1287 1287 E EGL_emulation: tid 1287: eglCreateSyncKHR(1669): error 0x3004
              (EGL_BAD_ATTRIBUTE)
            </div>
            <div className="log__warning">
              05-29 15:36:05.100 4065 4175 W NioEventLoop: Selector.select() returned prematurely 512 times in a row;
              rebuilding Selector io.netty.channel.nio.SelectedSelectionKeySetSelector@bac7eb3.
            </div>
            <div className="log__assert">
              05-29 15:36:06.100 4065 4175 A NioEventLoop: Selector.select() returned prematurely 512 times in a row;
              rebuilding Selector io.netty.channel.nio.SelectedSelectionKeySetSelector@bac7eb3.
            </div>
          </div>
```

**Comment by artem-zinnatullin** [comment]
I think this can be closed now, was done in #48.
